scripts/deploy.ts

```diff
@@ -521,6 +542,15 @@ async function coalesceWithPreviousAssets(assetsDir: string) {
 	}
 }
 
+async function packageAndPublishVSCodeExtension() {
+	const EXTENSION_DIR = './apps/vscode/extension'
+	await exec('yarn', ['version', 'patch'], { pwd: EXTENSION_DIR })
```

Comment:
As is, this version bump won't be persisted, and will end up being the same on subsequent deploys unless we manually bump it. Is that problematic for the marketplace? i.e. can we make multiple deploys with different code but with the same package.json version string?

Reply:
If the markeplace isn't cool with that there are two options I see:

1. fetch the latest deployed version from the marketplace api and bump that
2. persist the package.json change by pushing the commit to both `main` and `production`

We do something like 2 for the sdk deploys but it's a pain to set up so I'd recommend trying out number 1 first.